host-interaction/process/inject/inject-thread.yml

```diff
@@ -9,6 +9,8 @@ rule:
     att&ck:
       - Defense Evasion::Process Injection::Thread Execution Hijacking [T1055.003]
       - Defense Evasion::Reflective Code Loading [T1620]
+    mbc:
+      - Defense Evasion::Process Injection::Thread Execution Hijacking [E1055.003]
```

Comment:
This ID doesn't show up in the STIX data.

Reply:
Removed incorrect mapping

Reply:
Does this need to be removed here?